screen/Auth/AuthScreen.js

```diff
@@ -1,10 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { View, Text, StyleSheet } from 'react-native';
 
-const AuthScreen = (props) => {
+import * as Google from 'expo-auth-session/providers/google';
```

Comment:
`useAuthRequest`라는 메소드만 사용하므로, 아래와 같이 import해주면 좋을 것 같습니다.

```suggestion
import { useAuthRequest } from 'expo-auth-session/providers/google';
```

Reply:
구조분해 할당해서 사용하도록 꼭 기억하면서 사용하겠습니다!!